README.md

```diff
@@ -149,6 +149,11 @@ module "aws_account" {
     }
   }
 }
+## IAM Permissions Boundaries
+
+The module supports setting a Permission Boundary on the workspace user or role by passing down permissions_boundaries.workspace_boundary and permissions_boundaries.workspace_boundary_name. In case you want to reference a permission boundary that needs to be attached to every IAM role/user that will be created by the workspace role/user then you can create this permission boundary by specifying permissions_boundaries.workload_boundary and permissions_boundaries.workload_boundary_name.
```

Comment:
The module supports setting a Permission Boundary on the workspace user or role by passing down `permissions_boundaries.workspace_boundary` and `permissions_boundaries.workspace_boundary_name`. In case you want to reference a permission boundary that needs to be attached to every IAM role/user that will be created by the workspace role/user then you can create this permission boundary by specifying `permissions_boundaries.workload_boundary` and `permissions_boundaries.workload_boundary_name`.